### DISCUSSION TITLE
Is my KDBX file version 4 or higher?

### DISCUSSION THREAD

**Discussion by SuperSentient** [keepassxreboot/keepassxc#9412] [Q&A] [answered]
A few months ago I switched from Keepass2 version 2.45 on Ubuntu to KeepassXC PPA and I've saved changes to it at least a few times. And I'm wondering if my KDBX file is the latest version now or not?
Thanks in advance. :)

**Comment by droidmonkey** [comment, marked as answer]
Go to Database -> Database Security -> Encryption Tab -> Observe the KDBX version

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/f62f6e0b-9a51-4d0b-a779-a9ce8a723bde)

**Reply by SuperSentient** [reply]
It says version 3 should I change it?

**Reply by droidmonkey** [reply]
Yes you should 😄

**Reply by SuperSentient** [reply]
Thanks I did. :+1: :)
